nicegui/elements/chat_message.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  template: `
+    <q-chat-message ref="chat_message"
```

Comment:
I guess the `ref` is not needed and can be removed.

Reply:
Ok. I was just doing "monkey see, monkey do" based on the fileupload element....